packages/push/src/instruction/mod.rs

```diff
@@ -75,12 +76,12 @@ impl PushInstruction {
 
     #[must_use]
     pub fn push_int(i: i64) -> Self {
-        IntInstruction::Push(i).into()
+        IntInstruction::push(i).into()
     }
 
     #[must_use]
     pub fn push_float(f: OrderedFloat<f64>) -> Self {
-        FloatInstruction::Push(f).into()
+        FloatInstruction::push_ordered_float(f).into()
     }
```

Comment:
Does it make sense to just align these and do push_ordered_float and push_float?

Reply:
Hmm then let's just leave it for this pr.